### ISSUE TITLE
Investigate distributed clustering scheme for entity resolution

### ISSUE THREAD

**Issue by j143** [j143/systemds#21]
**What is proposed?**

link: https://dbs.uni-leipzig.de/en/publication/title/comparative_evaluation_of_distributed_clustering_schemes_for_multi_source_entity_resolution

**Images:**

![image](https://user-images.githubusercontent.com/53068787/89728835-d4751000-da4d-11ea-94fc-db069f2aed30.png)

![image](https://user-images.githubusercontent.com/53068787/89728855-eb1b6700-da4d-11ea-8ea8-d70061b8652a.png)

**TL;DR:**
entity resolution in distributed fashion
